### DISCUSSION TITLE
Add a "Not Interested" Option to Videos in Subscriptions

### DISCUSSION THREAD

**Discussion by lewxdev** [libre-tube/LibreTube#5708] [Ideas]
### Describe your suggested feature

Similar to how the current "Mark as watched" option can remove videos from any feed, this feature would add a "Not interested" option that removes videos from the feed, but doesn't mark the video as watched.

AC: Remove a video from any tab's feed without marking it as watched
1. See a video you don't want to watch
2. Long press to open context menu
3. Select "Not interested"
4. Watch as the video is hidden from the feed

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I will fill out all of the requested information in this form.

**Comment by akano12** [comment]
What's the problem with marking it as watched so it won't appear? Doesn't it serve the same purpose you want, which is removing a video from the feed?

**Comment by lewxdev** [comment]
> What's the problem with marking it as watched so it won't appear? Doesn't it serve the same purpose you want, which is removing a video from the feed?

@akano12 Marking a video as watched also adds it to the watch history. I often use the watch history to reference back to content I've watched. Marking as watched may be a fair workaround currently, but it clutters the watch history with videos that haven't actually been watched and would make it difficult to find specific content.

**Comment by Bnyro** [comment]
That's such an edge case feature that I don't aim to implement this unless there's a large amount of other people being interested in this too.
